src/components/Episode/Episode.jsx

```diff
@@ -1,14 +1,17 @@
 import React, { useContext } from "react";
 import { EpisodesContext as Episodes } from "COMPONENTS/Episodes/";
+import EpisodePreview from "COMPONENTS/EpisodePreview/";
 import { s } from "./";
 
 export default function Episode(props){
 
 	//CONTEXT
 	//---------------------------
-	const { 
-		activeIndex,
-		episodeCount: episodes
+	const {
+		get: {
+			episodeCount,
+			activeIndex
+		}
 	} = useContext(Episodes);
```

Comment:
Might as well do:
```
const { 
    episodeCount,
    activeIndex
} = useContext(Episodes).get;
```
here, as we're not pulling-out anything other than `get` from the context.

Reply:
Done.